tests/routes/webapp/patch_beneficiary_test.py

```diff
@@ -45,6 +45,56 @@ def when_changes_are_allowed(self, app):
             assert response.json["departementCode"] == user.departementCode
             assert user.departementCode == data["departementCode"]
 
+        @pytest.mark.usefixtures("db_session")
+        def when_updating_serialization(self, app):
+            # given
+            user = UserFactory(address="1 rue des machines")
+            data = {
+                "publicName": "plop",
+                "email": "new@email.com",
+                "postalCode": "93020",
+                "phoneNumber": "0612345678",
```

Comment:
```suggestion
                "phoneNumber": "1234567890",
```

Si ça fonctionne, comme ça cela nous évite d'avoir un "vrai" numéro